<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterWidget</class>
 <widget class="QWidget" name="UpdaterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpdaterWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>update_version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_update_version_LineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="m_textBrowser"/>
   </item>
   <item>
    <widget class="QPushButton" name="m_rstpushButton">
     <property name="text">
      <string>factory reset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_otapushButton">
     <property name="text">
      <string>ota</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_updatePushButton">
     <property name="text">
      <string>update</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
